maps/DeadSpace/ishimura.dm

```diff
@@ -6,12 +6,13 @@
 	name = "Ishimura"
 	full_name = "USG Ishimura"
 	path = "ishimura"
-	station_levels = list(1,2,3) // Add 4 to this once the EVA Deck is complete, and remove this comment.
-	contact_levels = list(1,2) // Add 3 to this once the EVA Deck is complete, and remove this comment.
+	station_levels = list(1,2) // Add 3 to this once the EVA Deck is complete, and remove this comment.
+	contact_levels = list(1,2,3,4) // Add 4 to this once the EVA Deck is complete, and remove this comment.
 	player_levels = list(1,2) // Add 3 to this once the EVA Deck is ocmplete, and remove this comment.
 	admin_levels = list(4) // Change this to 5 once the EVA Deck is complete.
+	sealed_levels = list(4) // No edge-of-map teleportation for players who hit the edge, to prevent shitlery.
```

Comment:
zlevel 4 is aegis VII ?

Reply:
zlevel 4 is admin / evac site. So transit and evac.

Reply:
the aegis VII level should be added to this

Reply:
Oh, you're right.